integration/cluster_test.go

```diff
@@ -416,3 +419,77 @@ func clusterMustProgress(t *testing.T, membs []*member) {
 		mcancel()
 	}
 }
+
+func TestTransferLeader(t *testing.T) {
```

Comment:
@heyitsanthony Or we can check the the leader of the members directly. Once the leader transfer finishes, the leader of the member should be updated.